index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
```

Comment:
but maybe we should keep existing behaviour and if `opts.fields` is set return raw doc.value from this function? I'm thinking maybe yes.